test/multiverse/suites/awssqs/awssqs_instrumentation_test.rb

```diff
@@ -0,0 +1,145 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+class AwssqsInstrumentationTest < Minitest::Test
+  def setup
+    Aws.config.update(stub_responses: true)
+    @stats_engine = NewRelic::Agent.instance.stats_engine
+  end
+
+  def teardown
+    harvest_span_events!
+    mocha_teardown
+    NewRelic::Agent.instance.stats_engine.clear_stats
```

Comment:
Does using the instance variable here help? 
```suggestion
    @stats_engine.clear_stats
```

Or if not, do we need the instance var on line 8?

Reply:
oh yeah i didn't add that, that's something that's done as part of the instrumentation scaffold thor command. this is the template.
```
class <%= @class_name %>InstrumentationTest < Minitest::Test
  def setup
    @stats_engine = NewRelic::Agent.instance.stats_engine
  end

  def teardown
    NewRelic::Agent.instance.stats_engine.clear_stats
  end

  # Add tests here
end
```
im not really sure why it's like that, but i can remove those from my tests since i'm not using them. Perhaps we should update the template if thats not something we want to do every time.

Reply:
removed https://github.com/newrelic/newrelic-ruby-agent/pull/2679/commits/278a3b509968facd0aa481d2241325dc37d9da56